### ISSUE TITLE
Images screenshot to pasteboard do not display preview in history

### ISSUE THREAD

**Issue by mikejongbloet** [p0deje/Maccy#412]
Hey there - I saw this ticket when searching to see if others had hit this issue...https://github.com/p0deje/Maccy/issues/191

I was recommended Maccy and am trying out the open source download. I noticed that whenever I copy an image to clipboard using Cmd+Ctrl+Shift+4, I get a blank line in the history. Pasting the image does work - it's simply the preview that does not work. 

- Copying text works and displays
- Copying images works and displays
- Screenshotting to pasteboard does not

In the screenshot below you see all three scenarios...
![Screen Shot 2022-06-08 at 14 36 24](https://user-images.githubusercontent.com/381085/172631969-abe3e824-53fb-45d3-bf4d-fd210e4aae6e.jpg)

The challenge is when I have more than one screenshot in my pasteboard history, now I have no idea which is which.

In the other ticket you requested the user to look at two directories:
- `~/Library/Containers/org.p0deje.Maccy/
- `~/Library/Application Support/org.p0deje.Maccy`

I've looked for both of these directories and can't locate either. Not in my `user/Library` path or in the root `Library` path. This folder structure comes up when I run AppCleaner on Maccy:
![image](https://user-images.githubusercontent.com/381085/172633546-826c2f23-498b-4f90-aac0-20904374008f.png)

I've also tried an uninstall and fresh install via homebrew, which didn;t fix the issue.

Any other ideas for troubleshooting this?

Thanks
Mike

**Comment by fabifabulousity** [comment]
Yes this happens when the scrennshot format is set to JPG, as well when I copy JPG images, e.g. from the Photos App.

**Comment by p0deje** [comment]
Then it will be fixed in the next release.

**Comment by mikejongbloet** [comment]
Great - thank you @p0deje
